gclient/src/node/result.rs

```diff
@@ -0,0 +1,9 @@
+#[derive(Debug, thiserror::Error)]
```

Comment:
I really see no reason in so many almost empty files. result mod / port mod may be just moved inside mod.rs